### ISSUE TITLE
Vault test suite fails converge

### ISSUE THREAD

**Issue by luckymike** [sensu/sensu-chef#401]
Haven't done any triage yet, but just saw this when running the test suites.
```
  NoMethodError
         -------------
         undefined method `chef_vault_item' for Sensu::Helpers:Class
         
         Cookbook Trace:
         ---------------
           /tmp/kitchen/cache/cookbooks/sensu/libraries/sensu_helpers.rb:48:in `data_bag_item'
           /tmp/kitchen/cache/cookbooks/sensu/recipes/rabbitmq.rb:88:in `from_file'
           /tmp/kitchen/cache/cookbooks/sensu-test/recipes/default.rb:37:in `from_file'
```

**Comment by cwjohnston** [comment]
I'm not able to reproduce the same error, but I am seeing indications that this suite is not functional due to incomplete/invalid test fixtures:

```   ================================================================================
         Recipe Compile Error in /tmp/kitchen/cache/cookbooks/sensu-test/recipes/default.rb
         ================================================================================

         ChefVault::Exceptions::SecretDecryption
         ---------------------------------------
         sensu/config is not encrypted with your public key.  Contact an administrator of the vault item to encrypt for you!

         Cookbook Trace:
         ---------------
           /tmp/kitchen/cache/cookbooks/chef-vault/libraries/helpers.rb:37:in `chef_vault_item'
           /tmp/kitchen/cache/cookbooks/sensu/libraries/sensu_helpers.rb:48:in `data_bag_item'
           /tmp/kitchen/cache/cookbooks/sensu/recipes/rabbitmq.rb:88:in `from_file'
           /tmp/kitchen/cache/cookbooks/sensu-test/recipes/default.rb:37:in `from_file'

         Relevant File Content:
         ----------------------
         /tmp/kitchen/cache/cookbooks/chef-vault/libraries/helpers.rb:

          30:    # @example
          31:    # item = chef_vault_item('secrets', 'bacon')
          32:    # log 'Yeah buddy!' if item['_default']['type']
          33:    # @param [String] bag Name of the data bag to load from.
          34:    # @param [String] id Identifier of the data bag item to load.
          35:    def chef_vault_item(bag, id)
          36:      if ChefVault::Item.vault?(bag, id)
          37>>       ChefVault::Item.load(bag, id)
          38:      elsif node['chef-vault']['databag_fallback']
          39:        Chef::DataBagItem.load(bag, id)
          40:      else
          41:        fail "Trying to load a regular data bag item #{id} from #{bag}, and databag_fallback is disabled"
          42:      end
          43:    end
          44:
          45:    # Helper method which provides an environment wrapper for a data bag.
          46:    # This allows for easy access to current environment secrets inside
```

Currently the fixture looks like this:
```
{
  "id": "config",
  "rabbitmq": {
    "encrypted_data": "",
    "password": "vaultpassword"
  }
}
```

So I can understand the reason for the error, but I'm not sure how to update the fixture.

**Comment by cwjohnston** [comment]
I think this is fixed in 9c2c09df31397d85bdca2aa644ac47b1956c833b
